src/cli.ts

```diff
@@ -10,6 +10,7 @@ let options = yargs
     .help('help', 'View help information about this tool.')
     .option('create-package', { type: 'boolean', defaultDescription: 'true', description: 'Creates a zip package. This setting is ignored when deploy is enabled.' })
     .option('source-map', { type: 'boolean', defaultDescription: 'false', description: 'Enables generating sourcemap files, which allow debugging tools to show the original source code while running the emitted files.' })
+    .option('allow-brighterscript-in-brightscript', { type: 'boolean', defaultDescription: 'false', description: 'Enables BrighterScript code om BrightScript files.' })
```

Comment:
```suggestion
    .option('allow-brighterscript-in-brightscript', { alias: 'allow-brighter-script-in-bright-script', type: 'boolean', defaultDescription: 'false', description: 'Allow brighterscript features (classes, interfaces, etc...) to be included in BrightScript (`.brs`) files, and force those files to be transpiled..' })
```

I agree with this name, but I think the cli will provide it as `allowBrighterscriptInBrightscript` instead of `allowBrighterScriptInBrightScript`, so the cli argument won't actually get applied. To work around that, we can use an alias which will sync both options.